nautobot/dcim/models/cables.py

```diff
@@ -335,7 +335,7 @@ class CablePath(BaseModel):
     )
     destination_id = models.UUIDField(blank=True, null=True)
     destination = GenericForeignKey(ct_field="destination_type", fk_field="destination_id")
-    # TODO: Profile filtering on this field if it could benefit from an index
+    # 2.0 TODO: Profile filtering on this field if it could benefit from an index
```

Comment:
Doesn't seem like a breaking (hence 2.0) change to me - what am I missing? Same question on the matching comments below.